### DISCUSSION TITLE
不知道哪里的问题

### DISCUSSION THREAD

**Discussion by fordeslorn** [hect0x7/JMComic-Crawler-Python#274] [Q&A] [answered]
<img width="995" alt="sp20240820_201903_006" src="https://github.com/user-attachments/assets/15f0d13f-0e76-4eba-9cae-91bc38b27a25">
代码没动一点，前几天还能用，今天再运行就失败了

**Comment by hect0x7** [comment]
> 
> <img width="995" alt="sp20240820_201903_006" src="https://github.com/user-attachments/assets/15f0d13f-0e76-4eba-9cae-91bc38b27a25">
> 代码没动一点，前几天还能用，今天再运行就失败了
> 

应该是禁漫的问题，禁漫服务器时不时炸

**Reply by fordeslorn** [reply]
好的，等下再试试😊

**Comment by hect0x7** [comment, marked as answer]
禁漫线路更新了，需要更新下jmcomic（v2.5.18）才能用
pip install jmcomic -i https://pypi.org/project -U

**Reply by fordeslorn** [reply]
可以了，感谢👍
